src/api/app/models/user.rb

```diff
@@ -840,6 +841,11 @@ def mark_login!
     update_attributes(last_logged_in_at: Time.now, login_failure_count: 0)
   end
 
+  def send_metric_for_beta_change
+    channel = (in_beta? ? 'joined_beta' : 'left_beta')
+    RabbitmqBus.send_to_bus('metrics', "user.#{channel} id=#{id}")
```

Comment:
Please try to avoid sending things that you can identify people with. It's enough to send the fact, then we can count/compare this.

Reply:
IIRC we need to track something (as a field). I took the state now. Could also be some random string though.